patient-care/webapp/client/shared/editSampleGroup.html

```diff
@@ -196,7 +196,7 @@
     </button>
   {{else}}
     {{#if length options.sample_labels}}
-      {{> listSamplesButton options.sample_labels}}
+      {{> listSamplesButton sampleLabels=options.sample_labels}}
```

Comment:
Note: we're not passing a filename, so it doesn't show a download button. When they see this list they're creating a new sample group so it'd be weird to have one.